tests/integration/test_system_detached_tables/test.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+from helpers.cluster import ClickHouseCluster
+
+cluster = ClickHouseCluster(__file__)
+node = cluster.add_instance("node_default", stay_alive=True)
+
+
+@pytest.fixture(scope="module", autouse=True)
+def start_cluster():
+    try:
+        cluster.start()
+        yield cluster
+    finally:
+        cluster.shutdown()
+
+
+def test_system_detached_tables():
```

Comment:
Could you please add similar tests to all supported database engines please?

Reply:
I add tests for Lazy and Replicated.
Let me know if need to add tests for another db engines (found here https://clickhouse.com/docs/en/sql-reference/statements/create/database#engine).